### DISCUSSION TITLE
PF14 Lightbox docs component Not Found

### DISCUSSION THREAD

**Discussion by aleon1220** [primefaces/community#1285] [PrimeFaces] [answered]
404 docs not found in PF14

https://primefaces.github.io/primefaces/14_0_0/#/components/lightbox

**Comment by melloware** [comment, marked as answer]
Yep check the 13.0.0 Migration guide: https://primefaces.github.io/primefaces/13_0_0/#/../migrationguide/13_0_0

You might also want to consult 14.0.0 Migration Guide if you are moving to 14: https://primefaces.github.io/primefaces/14_0_0/#/../migrationguide/14_0_0

**Reply by aleon1220** [reply]
thanks. I am using PF14 to test from scratch. When are you planning to release PrimeFaces 14 officially?

**Reply by melloware** [reply]
It should be pretty soon. 14.0.0-RC1 is very stable though for you to at least begin. And use Galleria if you were going to use Lightbox.

**Reply by aleon1220** [reply]
@melloware I am sorry i am new again to primeFaces. What do you mean Galleria? please post any links :-)

Thank you so much

**Reply by melloware** [reply]
See: https://www.primefaces.org/showcase/ui/multimedia/galleria/basic.xhtml?jfwid=3f07b
